internal/collector/pgadmin.go

```diff
@@ -21,24 +21,20 @@ func EnablePgAdminLogging(ctx context.Context, spec *v1beta1.InstrumentationSpec
 		return nil
 	}
 	otelConfig := NewConfig(spec)
-	otelConfig.Extensions["file_storage/pgadmin"] = map[string]any{
-		"directory":        "/var/log/pgadmin/receiver",
-		"create_directory": true,
-		"fsync":            true,
-	}
-	otelConfig.Extensions["file_storage/gunicorn"] = map[string]any{
-		"directory":        "/var/log/gunicorn" + "/receiver",
-		"create_directory": true,
+
+	otelConfig.Extensions["file_storage/pgadmin_data_logs"] = map[string]any{
+		"directory":        "/var/lib/pgadmin/logs/receiver",
+		"create_directory": false,
```

Comment:
OK, so we're creating the `/receiver` directory in pod.go

```
shell.MakeDirectories(0o775, dataMountPath, path.Join(LogDirectoryAbsolutePath, "receiver")),
```

hmmm, not sure I have a question here, just interested in the different way we're doing this.

Reply:
I've put TODOs elsewhere to create the directory using our function. This `create_directory` option does not do what we need in Kubernetes; [opentelemetry-collector-contrib#37774](https://www.github.com/open-telemetry/opentelemetry-collector-contrib/issues/37774)

IMV, this directory is a collector concern and belongs in its (the sidecar's) shell entrypoint or other init.